system/lib/wasmfs/js_api.cpp

```diff
@@ -169,4 +169,8 @@ void _wasmfs_readdir_finish(struct wasmfs_readdir_state* state) {
   free(state);
 }
 
+const char* __wasmfs_getcwd(){
```

Comment:
The convention here seems to be a single underscore prefix.

Reply:
Will rectify this. Thank you